app/tests/store.test.js

```diff
@@ -6,28 +6,9 @@ import { browserHistory } from 'react-router-dom';
 import configureStore from '../configureStore';
 
 describe('configureStore', () => {
-  let store;
-
-  beforeAll(() => {
-    store = configureStore({}, browserHistory);
-  });
-
-  describe('injectedReducers', () => {
-    it('should contain an object for reducers', () => {
-      expect(typeof store.injectedReducers).toBe('object');
-    });
-  });
-
-  describe('injectedSagas', () => {
-    it('should contain an object for sagas', () => {
-      expect(typeof store.injectedSagas).toBe('object');
-    });
-  });
-
-  describe('runSaga', () => {
-    it('should contain a hook for `sagaMiddleware.run`', () => {
-      expect(typeof store.runSaga).toBe('function');
-    });
+  it('should return a redux store', () => {
+    const store = configureStore({}, browserHistory);
+    expect(typeof store).toBe('object');
```

Comment:
It's a bit light to say it should return a redux store but then just check that it's of type object. You should use some of the same checks that used to be in checkStore. And IMO there's nothing wrong with also checking for the presence of the keys which are added by redux-injectors (runSaga, injectedSagas, injectedReducers, and replaceReducer too or is that gone now?).

Reply:
So I added in a more robust test for checking if it returned a redux store. I'm hesitant to check for `injectedSagas`, `injectedReducers`, etc, because I'd consider that an internal implementation detail of `redux-injectors`.  I'd like to be able to change that in the future without breaking `react-boilerplate` tests.